data/ecosystems/m/multiversx-elrond.toml

```diff
@@ -1239,3 +1239,54 @@ url = "https://github.com/yum0e/my-simple-explanations"
 
 [[repo]]
 url = "https://github.com/zdenal/elrond-amm"
+
+[[repo]]
+url = "https://github.com/validblocks/xSafe"
+
+[[repo]]
+url = "https://github.com/Catenscia/MxOps"
+
+[[repo]]
+url = "https://github.com/drdelphi/EsdtBingo"
+
+[[repo]]
+url = "https://github.com/damienen/reputation-system"
+
+[[repo]]
+url = "https://github.com/adrianlari/the-bucharest-hackathon"
+
+[[repo]]
+url = "https://github.com/cmalex23/hackathon2023/"
+
+[[repo]]
+url = "https://github.com/CoinDrip-finance/vesto-sc"
+
+[[repo]]
+url = "https://github.com/Awesome-Purple-Hackers"
+
+[[repo]]
+url = "https://github.com/mihaicalinluca/bucharest-hackaton"
+
+[[repo]]
+url = "https://github.com/mihaicherechesu/bucharest-hackathon-mmap"
+
+[[repo]]
+url = "https://github.com/andreigiura/ngx-sdk-dapp"
+
+[[repo]]
+url = "https://github.com/wdcurry/mvxConsole"
+
+[[repo]]
+url = "https://github.com/multiversx/mx-nft-verify-dapp"
+
+[[repo]]
+url = "https://github.com/esdt-space/mx-dapp-template"
+
+[[repo]]
+url = "https://github.com/Moonlorian/vesting-dapp"
+
+[[repo]]
+url = "https://github.com/cyberdeckstudios/WalletConnectV2_Demo"
+
+[[repo]]
+url = "https://github.com/cyberdeckstudios/pyWalletConnect"
```

Comment:
can you please point out how can I identify MultiversX within the code of `pyWalletConnect`?

Reply:
Solved

Reply:
hey @schimih, I'm somehow still seeing the pyWalletConnect repo, did you remove it from the list?

Reply:
@schimih any chance you can take a look at this so we can finalize this PR?